### PR TITLE
Feat/api buildsystem

### PR DESCRIPTION
**Pull request by henriq4**

### CODE REVIEW

File: apps/api/.npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+auto-install-peers=true
```
Comment (eduardoteles17) [inline apps/api/.npmrc:1]: This file must be in root of project

**Review by eduardoteles17 — CHANGES_REQUESTED** [review]
Just need to change the .npmrc to the project root

File: apps/api/.npmrc (hunk 1 above)
Comment (henriq4) [reply apps/api/.npmrc]: Done!

**Review by eduardoteles17 — APPROVED** [review]
ok

**Pull request merged**